data/core/macros/weapon_specials.cfg

```diff
@@ -94,20 +94,28 @@
     [/petrifies]
 #enddef
 
-#define WEAPON_SPECIAL_MARKSMAN
+#define WEAPON_SPECIAL_OFFENSE_PRECISION NAME SPECIAL_NOTES
     # Canned definition of the Marksman ability to be included in a
     # [specials] clause.
     [chance_to_hit]
         id=marksman
-        name= _ "marksman"
+        name={NAME}
         description= _ "When used offensively, this attack always has at least a 60% chance to hit."
-        special_note={INTERNAL:SPECIAL_NOTES_MARKSMAN}
+        special_note={SPECIAL_NOTES}
         value=60
         cumulative=yes
         active_on=offense
     [/chance_to_hit]
 #enddef
 
+#define WEAPON_SPECIAL_MARKSMAN
+    {WEAPON_SPECIAL_OFFENSE_PRECISION _"marksman" {INTERNAL:SPECIAL_NOTES_MARKSMAN}}
+#enddef
+
+#define WEAPON_SPECIAL_PRECISE_STRIKE
+    {WEAPON_SPECIAL_OFFENSE_PRECISION _"precise strike" {INTERNAL:SPECIAL_NOTES_PRECISE_STRIKE}}
```

Comment:
would it make sense to add translation comment that it may be translated same as marksman for languages where current was already fitting?